src/System.Net.Http/tests/StressTests/HttpStress/ChecksumHelpers.cs

```diff
@@ -65,6 +65,7 @@ public static ulong update_crc(ulong crc, string text, Encoding encoding = null)
         }
 
         public static ulong CalculateCRC(byte[] buf) => update_crc(InitialCrc, buf, buf.Length) ^ InitialCrc;
+        public static ulong CalculateCRC(string text, Encoding encoding = null) => update_crc(InitialCrc, text, encoding) ^ InitialCrc;
```

Comment:
It looks like this method isn't being used, is this intentional?

Reply:
What do you mean? It's being used in quite a few places.

Reply:
Oh, sorry, I meant the `encoding` part of it :)